Pod/Classes/JSBadgeView.m

```diff
@@ -301,19 +331,51 @@ - (void)drawRect:(CGRect)rect
         const CGFloat marginToDrawInside = [self marginToDrawInside];
         const CGRect rectToDraw = CGRectInset(rect, marginToDrawInside, marginToDrawInside);
         
-        UIBezierPath *borderPath = [UIBezierPath bezierPathWithRoundedRect:rectToDraw byRoundingCorners:(UIRectCorner)UIRectCornerAllCorners cornerRadii:CGSizeMake(JSBadgeViewCornerRadius, JSBadgeViewCornerRadius)];
+        UIBezierPath *borderPath;
```

Comment:
Before adding all these new features to the API, I believe this implementation would need to simplify. This class used `-drawRect:` back when it was doing complicated shadows and gradients. Now that it only supports "flat UI", there's no reason to continue doing this. Adding all this code just further complicates this implementation.
Given that I don't really have time to maintain this library, I will push back on these changes for that reason. I will however appreciate re-doing this view with AutoLayout, and without drawing with CoreGraphics instead.